packages/api-derive/src/balances/votingBalancesNominatorsFor.ts

```diff
@@ -3,19 +3,22 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Observable } from 'rxjs';
-import { map, switchMap } from 'rxjs/operators';
+import { switchMap } from 'rxjs/operators';
 import { ApiInterface$Rx } from '@polkadot/api/types';
+import { Vector } from '@polkadot/types/codec';
 import { AccountId } from '@polkadot/types/index';
 
 import { DerivedBalances } from '../types';
 import { drr } from '../util/drr';
 import { votingBalances } from './votingBalances';
 
 export function votingBalancesNominatorsFor (api: ApiInterface$Rx) {
-  return (accountId: AccountId | string): Observable<Array<DerivedBalances>> =>
-    (api.query.staking.nominatorsFor(accountId) as Observable<Array<AccountId | undefined | null> | undefined>).pipe(
-      map((nominators) => nominators || []),
-      switchMap(votingBalances(api)),
-      drr()
-    );
+  return (accountId: AccountId | string): Observable<Array<DerivedBalances>> => {
+    // tslint:disable-next-line
```

Comment:
Ummm... TSLint was updated, and this looks the same as the issue @amaurymartiny had - I'm guessing TSLint actually has a bug.